src/client/src/app/app-routing.module.ts

```diff
@@ -1,25 +1,23 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { LeaderboardComponent } from './components/leaderboard/leaderboard.component';
-import { PlayComponent } from './components/play/play.component';
 import { SignInComponent } from './components/sign-in/sign-in.component';
-import { GameComponent } from './pages/game/game.component';
 import { HomeComponent } from './pages/home/home.component';
 import { SignUpComponent } from './pages/sign-up/sign-up.component';
-import { GameInfoResolver } from './resolvers/game-info.resolver';
 
 const routes: Routes = [
-  {path: 'game', component: GameComponent, resolve: {gameInfo: GameInfoResolver}},
```

Comment:
again... keeping this line and maybe adding lazy loading to this path .. might be more of what we are looking for

Reply:
will take care of it on next issue